test/e2e/storage/external/external.go

```diff
@@ -20,6 +20,8 @@ import (
 	"flag"
 	"io/ioutil"
 
+	"k8s.io/kubernetes/test/e2e/framework/volume"
```

Comment:
better to move this import line to line 34 for the same import group.

Reply:
thoughtful 👍